src/test/java/karate-config.js

```diff
@@ -0,0 +1,18 @@
+function fn() {    
+  var env = karate.env; // get system property 'karate.env'
+  karate.log('karate.env system property was:', env);
+  if (!env) {
+    env = 'dev';
+  }
+  var config = {
+    env: env,
+	myVarName: 'someValue'
+  }
+  if (env == 'dev') {
+    // customize
+    // e.g. config.foo = 'bar';
+  } else if (env == 'e2e') {
+    // customize
```

Comment:
Can we will change to other environment and not e2e?, What other environment use to wolox?

Reply:
I made some changes in this file on #4 (altough its more like a template).
This PR was focused on just setting the folder structure for the maven project and some basic files (this is the default karate-config for the generated archetype) so I didn't make any changes at that time

Reply:
Yeah, I had seen yet, but i had post the comment yet :)
Thks for aclaration